glide.yaml

```diff
@@ -36,4 +36,3 @@ import:
   subpackages:
   - context
 - package: google.golang.org/grpc
-  version: 0d9891286aca15aeb2b0a73be9f5946c3cfefa85
```

Comment:
I removed the pin because it was conflicting with the version that etcd requires.